src/logger/loggerApi.c

```diff
@@ -1659,10 +1670,45 @@ int api_setScript(Serial *serial, const jsmntok_t *json)
                 API_SUCCESS : API_ERROR_SEVERE;
 }
 
-int api_runScript(Serial *serial, const jsmntok_t *json)
+int api_runScript(struct Serial *serial, const jsmntok_t *json)
 {
         lua_task_stop();
         lua_task_start();
         return API_SUCCESS;
 }
 #endif /* LUA_SUPPORT */
+
+int api_set_wifi_client_cfg(struct Serial *s, const jsmntok_t *json)
```

Comment:
Not yet.  I have yet to write the code for the server.  Need to implement the device commands, then tie them into the driver.